### PR TITLE
Modernize documentation examples

### PR DESCRIPTION
**Pull request by bigtimebuddy**
This is all documentation changes, no code changes.

### Fixed

* Updates any old Loader examples to use Assets instead
* Added `import` statements to examples
* De-emphasis `PIXI` global namespace in favor of ESM syntax
* Converts a bunch of inline code blocks to `@example` syntax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,38 +121,38 @@ before submitting changes.
...
+bunny.anchor.y = 0.5;
+
+// Add the bunny to the scene we are building
+app.stage.addChild(bunny);
+
+// Listen for frame updates
+app.ticker.add(() => {
+      // each frame we spin the bunny around a bit
+    bunny.rotation += 0.01;
 });
```
Comment (SuperSodaSea) [inline README.md:156]: The indent seems to be incorrect, and also incorrect in the same code in bundles/(pixi.js/pixi.js-legacy/pixi.js-node)/README.md

Comment (bigtimebuddy) [reply README.md:156]: Good catch, thanks!

File: README.md [hunk 2]
```diff
@@ -121,38 +121,38 @@ before submitting changes.
...
-
-    // Add the bunny to the scene we are building
-    app.stage.addChild(bunny);
-
-    // Listen for frame updates
-    app.ticker.add(() => {
-         // each frame we spin the bunny around a bit
-        bunny.rotation += 0.01;
-    });
+const texture = await Assets.load('bunny.png');
```
Comment (domis86) [inline README.md:136]: `Assets` seem not imported in this example (and few others added in this PR).

Comment (bigtimebuddy) [reply README.md:136]: Good catch @domis86. Will fix

**Pull request merged**